draft-packages/filter-menu-button/docs/FilterMenuButton.stories.tsx

```diff
@@ -242,3 +242,163 @@ export const DefaultWithChildrenAdvancedFilter = () => {
 DefaultWithChildrenAdvancedFilter.storyName =
   "Default with children (Advanced filter)"
 DefaultWithChildrenAdvancedFilter.parameters = { chromatic: { disable: false } }
+
+export const AutoHideBehaviours = () => {
+  const [isDropdownVisible1, setIsDropdownVisible1] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [isDropdownVisible2, setIsDropdownVisible2] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [isDropdownVisible3, setIsDropdownVisible3] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [appliedFilters, setAppliedFilters] = useState<DropdownOption[]>(
+    IS_CHROMATIC ? DROPDOWN_OPTIONS_CHROMATIC_SELECTED : []
+  )
+
+  const checkedTraits: string = appliedFilters
+    .map(trait => trait.label)
+    .join(", ")
+
+  const toggleDropdown1 = () => setIsDropdownVisible1(!isDropdownVisible1)
+
+  const hideDropdown1 = () => setIsDropdownVisible1(false)
+
+  const toggleDropdown2 = () => setIsDropdownVisible2(!isDropdownVisible2)
+
+  const hideDropdown2 = () => setIsDropdownVisible2(false)
+
+  const toggleDropdown3 = () => setIsDropdownVisible3(!isDropdownVisible3)
+
+  const hideDropdown3 = () => setIsDropdownVisible3(false)
+
+  const onCheckboxChange = (option: DropdownOption) => {
+    if (appliedFilters.find(filter => filter.id === option.id)) {
+      setAppliedFilters(
+        appliedFilters.filter(filter => filter.id !== option.id)
+      )
+    } else {
+      setAppliedFilters([...appliedFilters, option])
+    }
+  }
+
+  return (
+    <>
+      <FilterMenuButton
+        id={"filter-menu-button--simple1"}
+        labelText={"Animal traits autoHide on"}
+        metadata={checkedTraits}
+        isDropdownVisible={isDropdownVisible1}
+        toggleDropdown={toggleDropdown1}
+        hideDropdown={hideDropdown1}
+        autoHide="on"
+      >
+        <>
+          <div className={styles.content}>
+            <CheckboxGroup labelText="Traits">
+              {DROPDOWN_OPTIONS.map(trait => (
```

Comment:
When I select any option in any of the 3 dropdowns, the selected values apply to all 3 buttons as opposed to just the one I chose from.
Since the contents of the menu don't matter as long as there's at least one option, you could simplify and just hard code one option per each.